js_modules/dagit/packages/core/src/partitions/PartitionRunMatrix.tsx

```diff
@@ -242,9 +244,9 @@ export const PartitionRunMatrix: React.FC<PartitionRunMatrixProps> = (props) =>
               <MenuItem
                 text="Only show failures and gaps"
                 icon={
-                  <Icon
-                    icon="tick"
-                    color={options.showFailuresAndGapsOnly ? Colors.GRAY1 : Colors.LIGHT_GRAY3}
+                  <IconWIP
+                    name="done"
+                    color={options.showFailuresAndGapsOnly ? ColorsWIP.Gray700 : ColorsWIP.Gray200}
```

Comment:
I think right now the disabled WIP Button component is `Gray300`, I wonder if this should be too?

Reply:
I was really all over the place on these icon grays...